src/chains/ethereum/ethereum/tests/transaction-pool.test.ts

```diff
@@ -81,8 +81,12 @@ describe("transaction pool", async () => {
     // returns an account's nonce
     blockchain = {
```

Comment:
It kinda seems like we should add a test for the insufficient balance rejection here, right? To get all code paths

Reply:
added!